core/store/models/log_events.go

```diff
@@ -149,8 +156,6 @@ func FilterQueryFactory(i Initiator, from *big.Int) (q ethereum.FilterQuery, err
 		// [][]common.Hash) clarifies their type for reflect.DeepEqual
 		q.Topics = make([][]common.Hash, len(i.Topics))
 		copy(q.Topics, i.Topics)
-	case i.Type == InitiatorFluxMonitor:
```

Comment:
Per a conversation with @se3000, We no longer use `FilterQueryFactory` to handle FluxAggregator log queries.